packages/ubuntu_wsl_setup/lib/main.dart

```diff
@@ -56,6 +56,10 @@ Future<void> main(List<String> args) async {
     variant.value = value;
     if (value == Variant.WSL_SETUP) {
       subiquityMonitor.onStatusChanged.listen((status) {
+        if (status?.state == ApplicationState.EXITED ||
+            status?.state == ApplicationState.DONE) {
+          subiquityMonitor.stop();
+        }
```

Comment:
could be done in SubiquityStatusMonitor itself? (and a separate PR :)

Reply:
Done as well.